busybox: move /bin/su to /usr/bin/su to match util-linux and shadow

Both of util-linux and shadow have su binary in /usr/bin, fix busybox
to use the same path so they can be properly tracked by alternatives.

Upstream-Status: Inappropriate [embedded]
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 include/applets.src.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/applets.src.h b/include/applets.src.h
--- a/include/applets.src.h
+++ b/include/applets.src.h
@@ -349,7 +349,7 @@ IF_STAT(APPLET(stat, BB_DIR_USR_BIN, BB_SUID_DROP))
 IF_STRINGS(APPLET(strings, BB_DIR_USR_BIN, BB_SUID_DROP))
 IF_STTY(APPLET(stty, BB_DIR_BIN, BB_SUID_DROP))
 /* Needs to be run by root or be suid root - needs to change uid and gid: */
-IF_SU(APPLET(su, BB_DIR_BIN, BB_SUID_REQUIRE))
+IF_SU(APPLET(su, BB_DIR_USR_BIN, BB_SUID_REQUIRE))
 IF_SULOGIN(APPLET(sulogin, BB_DIR_SBIN, BB_SUID_DROP))
 IF_SUM(APPLET(sum, BB_DIR_USR_BIN, BB_SUID_DROP))
 IF_SV(APPLET(sv, BB_DIR_USR_BIN, BB_SUID_DROP))
-- 
1.8.1.2

